netconf/transport.go

```diff
@@ -74,9 +74,12 @@ func (t *transportBasicIO) Send(data []byte) error {
 	}
 	dataInfo = append(dataInfo, data...)
 	dataInfo = append(dataInfo, seperator...)
-	t.Write(dataInfo)
+	_, err := t.Write(dataInfo)
+	if err != nil {
```

Comment:
remove the if block and just since if it is nil we will return it anyways
return err

```
_, err := t.Write(dataInfo)
return err
```

Reply:
Good catch
Fixed